docs/en/stack/ml/nlp/ml-nlp-model-ref.asciidoc

```diff
@@ -97,3 +185,27 @@ Using `DPREncoderWrapper`:
 * https://huggingface.co/valhalla/distilbart-mnli-12-6
 * https://huggingface.co/cross-encoder/nli-distilroberta-base
 * https://huggingface.co/cross-encoder/nli-roberta-base
+
+[discrete]
+=== Zero-shot text classification expected model output
+
+Zero-shot text classification allows text to be classified for arbitrary labels not necessarily part of the original
+training. Each sequence is combined with the label given some hypothesis template. The model then scores each of these
+combinations according to `[entailment, neutral, contradiction]`
+
+The required output of the model is <number of sequences> <number of labels> <classification for label>.
```

Comment:
> The required output of the model is <number of sequences> <number of labels> <classification for label>.

Per my suggestion for the previous sections, I suggest using something like this instead: "The required output of the model is a nested set of {arrays? objects? tuples?} for each inferred sequence, where every sequence has "entailment", "neutral", and "contradiction" classification scores for each (entity?) label."